### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SaeqaSultani**
-     Select the API you are going to use.
-     Build files structure for your React app.
-     Prepare routes and navigation in your `app.`
-     Make sure that a user can display a list of items and filter them. Data should come from the API.
-     Create the tests for the application.
-     Style your components to match the design provided.
-     Deploy the project and test for final details.
-     Record a video for your project.
-     Create a good README and PR description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- npm run test
+
+# Video Link :)
+
+[Video Link](https://www.loom.com/share/7ec36e12e80341e295598cdb901ae8d2)
+
+
+# Live Demo :)
+
+[Live Demo Link](https://saeqasultani.github.io/capstone-project-/)
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly provide the correct link to your deployed app for this project. This link is for your HTML capstone project.

  der

File: src/components/details/Detailspage.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import './Detailspage.css';
+
+const Detailspage = () => {
+  const { companyDetails } = useSelector((state) => state.companies);
+  const dispatch = useDispatch();
+  const { smbl } = useParams();
+  useEffect(() => {
+    dispatch(getCompanyDetails(smbl));
+  }, [dispatch]);
+  // console.log(`companyDetails ${companyDetails}`);
```
Comment (TedLivist) [inline src/components/details/Detailspage.js]: - [ ] Kindly remove commented code.

File: src/redux/companies-list.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    dispatch({ type: GET_COMPANIES, companies: data });
+  },
+);
+
+export const getCompanyDetails = createAsyncThunk(
+  GET_COMPANIEY_DETAILS,
+  async (args, { dispatch }) => {
+    const response = await fetch(`https://financialmodelingprep.com/api/v3/profile/${args}?apikey=6f7d6f7fb9d6628c3bcdc2ea4a338d9f`);
+    const data = await response.json();
+    // console.log(`data: ${data}`);
```
Comment (TedLivist) [inline src/redux/companies-list.js:43]: - [ ] Kindly remove commented code.

File: src/components/homepage/HomepageList.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            aria-hidden="true"
+          >
+            <HomepageItem title={company.symbol} />
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
+
```
Comment (TedLivist) [inline src/components/homepage/HomepageList.js]: - [ ] Based on the requirements there should be a search or filter mechanism on the home page that allows the users to filter items on the home page. Kindly implement that.

  furkan-filter-2

File: src/components/navbar/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+      <div key={uuidv4()} onClick={() => navigate('/')} aria-hidden="true">
+        <img
+          className="btn"
+          src={back}
+          alt="back"
+        />
+      </div>
+
+      <h2 className="heading">Componies</h2>
```
Comment (TedLivist) [inline src/components/navbar/Navbar.js:22]: - [ ] **[OPTIONAL]** I think this should be **companies**, with "a"

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @SaeqaSultani,

Good job so far!
✔️ Good documentation
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+- npm run test
+
+# Video Link :)
+
+[Video Link](https://www.loom.com/share/7ec36e12e80341e295598cdb901ae8d2)
+
+
+# Live Demo :)
+
+[Live Demo Link](https://react-capstone-project-metrics-webapp.netlify.app/)
```
Comment (sja-thedude) [inline README.md]: - [ ] Your link is not working, It is not showing the deployed version of your project.

File: src/components/details/Detailspage.js (hunk 2 above)
Comment (sja-thedude) [reply src/components/details/Detailspage.js]: - [ ] Kindly make sure to remove the commented code.

File: src/redux/companies-list.js (hunk 3 above)
Comment (sja-thedude) [reply src/redux/companies-list.js:43]: - [ ] Kindly make sure to remove the commented code.

File: src/components/homepage/HomepageList.js (hunk 4 above)
Comment (sja-thedude) [reply src/components/homepage/HomepageList.js]: - [ ] As per the previous reviewer kindly add a search or filter mechanism on the home page that allows the users to filter items on the home page. Kindly implement that.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @SaeqaSultani, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @SaeqaSultani , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good documentation
- Good code structure
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**